app/controllers/concerns/eventable.rb

```diff
@@ -6,6 +6,10 @@ def supplier_id
     @supplier_id ||= current_user.owner&.id
   end
 
+  def created_by
+    user_for_paper_trail || current_user&.owner&.name || 'unknown'
```

Comment:
since we are adding the created by to all events, is there a shared example we can add a test to, to verify this behaviour?

Reply:
Unsure there is a shared example, but I think I've covered all places where the behaviour header -> api controller -> individual controller -> model is tested.